clang/test/APINotes/Inputs/Headers/SwiftImportAs.apinotes

```diff
@@ -3,6 +3,12 @@ Name: SwiftImportAs
 Tags:
 - Name: ImmortalRefType
   SwiftImportAs: reference
+  Methods:
+    - Name: methodReturningFrt__
+    - Name: methodReturningFrt_returns_unretained
+      SwiftReturnOwnership: 'returns_unretained'
+    - Name: methodReturningFrt_returns_retained
+      SwiftReturnOwnership: 'returns_retained'
```

Comment:
We shouldn't require wrapping the attribute in quotes – quotes aren't used for existing attributes.